src/sentry/event_manager.py

```diff
@@ -787,39 +811,44 @@ def save(self, project_id, raw=False, assume_normalized=False):
             #         group.id: 1,
             #     },
             # })
-            (tsdb.models.frequent_environments_by_group, {
-                group.id: {
-                    environment.id: 1,
-                },
-            })
         ]
 
-        if release:
+        if group:
             frequencies.append(
-                (tsdb.models.frequent_releases_by_group, {
+                (tsdb.models.frequent_environments_by_group, {
                     group.id: {
-                        grouprelease.id: 1,
+                        environment.id: 1,
                     },
                 })
             )
 
-        tsdb.record_frequency_multi(frequencies, timestamp=event.datetime)
+            if release:
+                frequencies.append(
+                    (tsdb.models.frequent_releases_by_group, {
+                        group.id: {
+                            grouprelease.id: 1,
+                        },
+                    })
+                )
+        if frequencies:
```

Comment:
We may want to add some frequencies stats for issueless events.

Reply:
Would we need a way to identify different transactions to do that?